t/gh323.t

```diff
@@ -2,6 +2,10 @@ use Test2::V0 -no_srand => 1;
 use FFI::Platypus;
 use FFI::Platypus::Memory qw( malloc free );
 
+plan skip_all => 'test requires variadic function support'
```

Comment:
```suggestion
skip_all 'test requires variadic function support'
```
:point_up_2: Test2 plan only takes a count https://metacpan.org/pod/Test2::V0#plan($count)
skip_all is a function https://metacpan.org/pod/Test2::V0#skip_all($reason)

Reply:
Interestingly the current version does accept a skip_all presumably for backward compatibility
https://metacpan.org/module/Test2::Tools::Basic/source#L85